.projenrc.ts

```diff
@@ -14,6 +16,66 @@ const project = new CdklabsConstructLibrary({
     secret: 'GITHUB_TOKEN',
   },
   autoApproveUpgrades: true,
+  prettier: true,
+  gitignore: [
+    '*.d.ts',
+    '*.generated.ts',
+    '*.js',
+    '**/integ.*.snapshot/asset.*/',
+    '**/bin',
+    '**/obj',
+    '/*.sln',
+  ],
+  publishToPypi: {
+    distName: 'cdklabs.aws-lambda-dotnet',
+    module: 'cdklabs.aws_lambda_dotnet',
+  },
+  publishToNuget: {
+    packageId: 'Cdklabs.AWS.Lambda.DotNet',
+    dotNetNamespace: 'Cdklabs.AWS.Lambda.DotNet',
+  },
+  publishToMaven: {
+    mavenGroupId: 'io.github.cdklabs',
+    javaPackage: 'io.github.cdklabs.aws.lambda.dotnet',
+    mavenArtifactId: 'cdklabs-aws-lambda-dotnet',
+    mavenEndpoint: 'https://s01.oss.sonatype.org',
+  },
+  publishToGo: {
+    moduleName: 'github.com/cdklabs/awscdk-lambda-dotnet',
+  },
```

Comment:
```suggestion
  publishToPypi: {
    distName: 'aws-cdk.aws-lambda-dotnet',
    module: 'aws_cdk.aws_lambda_dotnet',
  },
  publishToNuget: {
    packageId: 'Amazon.CDK.Lambda.DotNet',
    dotNetNamespace: 'Amazon.CDK.Lambda.DotNet',
  },
  publishToMaven: {
    mavenGroupId: 'software.amazon.awscdk',
    javaPackage: 'software.amazon.awscdk.cdk.lambda.dotnet',
    mavenArtifactId: 'aws-lambda-dotnet',
    mavenEndpoint: 'https://aws.oss.sonatype.org',
  },
```

No go publishing because that requires another repo, `cdklabs/awscdk-lambda-dotnet-go`, and I don't think it's worth it for now.